research/src/pages/components/selectmenu.mdx

```diff
@@ -0,0 +1,257 @@
+---
```

Comment:
You'll also need to add the menu section you want this displayed under:

```
menu: Research
```

Reply:
I did "proposals" since I want it to show up in the proposals section in the sidebar